Tests/BootCamp.Chapter.Tests/HandTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BootCamp.Chapter.Gambling;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class HandTest
+    {
+        // The constructor cannot be tested because cards is protected. 
+
+        // AddCards cannot be tested because the return type is void and Cards cannot be read outside the class. 
```

Comment:
Yes, but there is a `Reveal` method, through which you can get all the cards. Not always the result of calling a function is a direct output. Sometimes you need to check the effects of the system under test through additional functions, like in this case- Reveal.

Reply:
Thanks, also changed